modules/rostests/kmtests/include/kmt_test.h

```diff
@@ -233,7 +233,7 @@ VOID KmtFreeGuarded(PVOID Pointer);
 #ifdef KMT_KERNEL_MODE
 #define ok_irql(irql)                       ok(KeGetCurrentIrql() == irql, "IRQL is %d, expected %d\n", KeGetCurrentIrql(), irql)
 #endif /* defined KMT_KERNEL_MODE */
-#define ok_eq_print(value, expected, spec)  ok((value) == (expected), #value " = " spec ", expected " spec "\n", value, expected)
+#define ok_eq_print(value, expected, spec)  ok((value) == (expected), #value " = " spec ", expected " #expected "(" spec ")\n", value, expected)
```

Comment:
`spec` is now evaluated twice?

Reply:
No, it just adds the expected value as a string as it was passed to the macro. Well it was supposed to, but in fact it doesn't help much, as this macro is used from another macro, which already evaluates it. I'll try to improve it.